vm/vm.go

```diff
@@ -157,35 +157,74 @@ func (mod) hex() []uint8 {
 	return []uint8{uint8(opcode.Mod)}
 }
 
-// TODO: implement me w/ test cases :-)
 func (lt) Do(stack *stack, _ asmReader) error {
+	y := stack.pop()
+	x := stack.pop()
+
+	Y := int(y)
+	X := int(x)
+
+	if X < Y {
+		stack.push(item(1))
+	} else {
+		stack.push(item(0))
+	}
+
 	return nil
 }
 
 func (lt) hex() []uint8 {
 	return []uint8{uint8(opcode.LT)}
 }
 
-// TODO: implement me w/ test cases :-)
 func (gt) Do(stack *stack, _ asmReader) error {
+	y := stack.pop()
+	x := stack.pop()
+
+	Y := int(y)
+	X := int(x)
+
+	if X > Y {
+		stack.push(item(1))
+	} else {
+		stack.push(item(0))
+	}
+
 	return nil
 }
 
 func (gt) hex() []uint8 {
 	return []uint8{uint8(opcode.GT)}
 }
 
-// TODO: implement me w/ test cases :-)
 func (eq) Do(stack *stack, _ asmReader) error {
+	y := stack.pop()
+	x := stack.pop()
+
+	Y := int(y)
+	X := int(x)
```

Comment:
There are 3 code blocks exactly same as above 4 lines.
How about use function to reduce duplicate codes?

Reply:
thanks. maybe that capital X and small x could be vague naming.
so how about change into this form. I referenced go-ethereum code.
```
x, y := int(stack.pop()), int(stack.pop())
```
@leesd556